odoo/addons/base/migrations/12.0.1.3/noupdate_changes.xml

```diff
@@ -1,17 +1,172 @@
 <?xml version='1.0' encoding='utf-8'?>
 <odoo>
-  <record id="gt" model="res.country">
-    <field name="currency_id" ref="GTQ"/>
+  <record id="cc" model="res.country">
+    <field eval="61" name="phone_code"/>
   </record>
-  <record id="sg" model="res.country">
-    <field name="vat_label">GST No.</field>
+  <record id="aq" model="res.country">
+    <field eval="672" name="phone_code"/>
+  </record>
+  <record id="kn" model="res.country">
+    <field eval="1869" name="phone_code"/>
+  </record>
+  <record id="mp" model="res.country">
+    <field eval="1670" name="phone_code"/>
+  </record>
+  <record id="eh" model="res.country">
+    <field eval="212" name="phone_code"/>
+  </record>
+  <record id="do" model="res.country">
+    <field eval="1849" name="phone_code"/>
+  </record>
+  <record id="gu" model="res.country">
+    <field eval="1671" name="phone_code"/>
+  </record>
+  <record id="nf" model="res.country">
+    <field eval="672" name="phone_code"/>
+  </record>
+  <record id="pn" model="res.country">
+    <field eval="64" name="phone_code"/>
+  </record>
+  <record id="sx" model="res.country">
+    <field eval="1721" name="phone_code"/>
+  </record>
+  <record id="ax" model="res.country">
+    <field eval="358" name="phone_code"/>
+  </record>
+  <record id="im" model="res.country">
+    <field eval="44" name="phone_code"/>
+  </record>
+  <record id="bm" model="res.country">
+    <field eval="1441" name="phone_code"/>
+  </record>
+  <record id="lc" model="res.country">
+    <field eval="1758" name="phone_code"/>
+  </record>
+  <record id="bq" model="res.country">
+    <field eval="599" name="phone_code"/>
+  </record>
+  <!--
+  <record id="user_root" model="res.users">
```

Comment:
@pedrobaeza Should I decomment also this one?

Reply:
What the signature contains?